src/textarea.story.jsx

```diff
@@ -0,0 +1,45 @@
+
+import React from 'react';
+import Textarea from './Textarea';
+import { storiesOf } from '@kadira/storybook';
+
+storiesOf('Textarea', module)
+	.add('default', () => <Textarea
```

Comment:
dont forget to use the addInfo so we get some of the nice doc's for the component